### DISCUSSION TITLE
Is AtomWithHash deprecated in V2

### DISCUSSION THREAD

**Discussion by michael1997** [pmndrs/jotai#1762] [Q&A] [answered]
I'm looking to migrate to v2. We use AtomWithHash quite extensively and it doesn't seem to be in the current release?
Is it planned to be released or needed to be bolted on?

**Comment by michael1997** [comment, marked as answer]
I've just found that it is moved to `jotai-location`. 

`import { atomWithHash } from 'jotai-location'`

**Reply by dai-shi** [reply]
Yeah, hopefully this answer ☝️ will help others.

Somebody asked it before https://github.com/pmndrs/jotai/discussions/1514#discussioncomment-4857666, so it's probably confusing to quite a few people.
